composer.json

```diff
@@ -17,7 +17,7 @@
         "php": "^7.2 || ^8.0",
         "ext-json": "*",
         "doctrine/annotations": "^1.10.2",
-        "liip/metadata-parser": "^0.2 || ^0.3 || ^0.4 || ^0.5",
+        "liip/metadata-parser": "dev-master as 0.5",
```

Comment:
note to self: change to 0.6 before merging.

Reply:
I adjusted the version and amended it to the commit 2ea133381e5d75d52303458ff30bdb9be974ec5a